packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -76,6 +124,30 @@ export class Table extends Construct {
         return this;
     }
 
+    public addReadAutoScaling(props: DynamoDBAutoScalingProps) {
```

Comment:
Seems like these can also be optionally defined declaratively in `TableProps`:

```
new Table(this, 'MyTable', {
  readAutoScaling: { ... },
  writeAutoScaling: { ... },
}
```

Reply:
done